selfdrive/car/__init__.py

```diff
@@ -132,6 +132,21 @@ def apply_std_steer_angle_limits(apply_angle, apply_angle_last, v_ego, LIMITS):
   return clip(apply_angle, apply_angle_last - angle_rate_lim, apply_angle_last + angle_rate_lim)
 
 
+def common_fault_avoidance(measured_value, max_value, request, current_request_frames=0, max_request_frames=1, cut_request_frames=1):
+  # Count up to max_request_frames, at which point we need to cut the request for cut_request_frames to avoid a fault
```

Comment:
in a """ comment, can you explain how this workaround works? Roughly, the car will hold your torque for a short amount of time if you set the request bit low, and max angle and max angle rate limits are reset when the request bit is low, so we can pulse the bit when we detect we're about to fault

Reply:
done